auth/api/grpc/endpoint.go

```diff
@@ -62,17 +62,26 @@ func authorizeEndpoint(svc auth.Service) endpoint.Endpoint {
 			return authorizeRes{}, err
 		}
 
-		_, err := svc.Identify(ctx, req.token)
+		authorized, err := svc.Authorize(ctx, req.Sub, req.Obj, req.Act)
 		if err != nil {
 			return authorizeRes{}, err
```

Comment:
No need for `authorizeRes{authorized: false}` ? Like [here](https://github.com/mainflux/mainflux/pull/1440/files#diff-8fcf53605bc54615ea033c69bbba7d42b516e64c236776073c358c3b420e9189R170) 
If you are considering it as the default value for bool, that's ok just make it consistent.